dts/bindings/led/dac-leds-common.yaml

```
@@ -0,0 +1,24 @@
+# Copyright (c) 2025, Prevas A/S
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Common fields for DAC LEDs
+
+child-binding:
+  description: Common DAC LED child node
+  properties:
+    dac:
```

```suggestion
    dac-dev:
```

I will choose this property name.